docsource/conf.py

```diff
@@ -52,7 +52,7 @@
 # The short X.Y version.
 version = '0.9'
 # The full version, including alpha/beta/rc tags.
-release = '0.9.0'
+release = '0.9.1'
 
```

Comment:
Should this be `-dev1`

Reply:
nah, not in the documentation config file.